### PR TITLE
Update drush-versions.md

### PR DESCRIPTION
**Pull request by fusionx1**
Closes #

## Effect
PR includes the following changes:
-
-
-

## Remaining Work
- [ ] List any outstanding todos
- [ ] If needed

### CODE REVIEW

**Comment by alexfornuto** [comment]
@fusionx1 please use the PR template to describe the change you're making.

File: source/_docs/drush-versions.md [hunk 1]
```diff
@@ -63,6 +63,15 @@ If you need to specify a minor version or a version not available on the platfor
 
 For more information, see [Avoiding “Dependency Hell” with Site-Local Drush](https://pantheon.io/blog/avoiding-dependency-hell-site-local-drush){.external}.
 
+## Upgrade you local drush 8.x-dev
+When working on drupal 8.4.x make sure to upgrade your composer to 8.x.dev to avoid errors/interruptions especially when using drush utilities and migration using migrate commands via drush.
```
Comment (alexfornuto) [inline source/_docs/drush-versions.md]: why `dev` specifically? Should we be suggesting customers use a `dev` version on their local computer?

**Comment by fusionx1** [comment]
Hey bro, 8.x.dev releases use the latest version that fixes the issues on drupal 8.4.x, last time I check it uses drush 8.16. Been using it for the last 4 months now and i haven't encountered issues on drush.

**Comment by alexfornuto** [comment]
@fusionx1 I discussed this PR with @rachelwhitton, and we decided to go in this direction for a couple reasons:

1) Since Drush versions are discussed at the opener of the doc, we didn't want to duplicate content.
2) Composer commands aren't useful for those not using Composer, and partial Composer usage, especially with D8, can get tricky quickly.

Please take a look and let us know if this is satisfactory.

**Comment by rachelwhitton** [comment]
+1

**Comment by fusionx1** [comment]
Hi Alex sorry for my late response. Caught up with QBR automation development, yes it works on 8.1.15, this pr is intended for users who are heavily using drush and using drupal 8.4.3 and up, I was personally working on a project that i heavily use drush and by following the current document that time wasn't that much of a help when i was figuring out how to fix my drush.

**Comment by rachelwhitton** [comment]
+1

**Pull request merged**